### ISSUE TITLE
aria-valuenow error

### ISSUE THREAD

**Issue by alexroseb** [videojs/video.js#4960]
## Description
Using the browser plugin ChromeLens, which is built on top of Chrome accessibility tools, there is an error "ARIA state and property values must be valid [AX_ARIA_04](https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_aria_04)". This points to the line:

`<div tabindex="0" class="vjs-progress-holder vjs-slider vjs-slider-horizontal" role="slider" aria-valuenow="NaN" aria-valuemin="0" aria-valuemax="100" aria-label="video progress bar" aria-valuetext="0:00">` 

The error is that ["NaN" is not an acceptable value for aria-valuenow](https://www.w3.org/TR/wai-aria-1.1/#aria-valuenow). 

This is reproduced in the [starter template test case](http://output.jsbin.com/axedog). 

## Steps to reproduce
1. Run ChromeLens on any page with a video.js player

### Expected
No ARIA errors

### Error output
"ARIA state and property values must be valid [AX_ARIA_04](https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_aria_04)" for every video player on the page.

## Suggested Fix
Ideally, aria-valuenow would initially be set to 0. It's being set to "NaN" because the slider doesn't appear yet due to the big play button, but it should be an easy change for it to start set at "0".

## Additional Information
Please include any additional information necessary here. Including the following:
#### videojs
4.11, 5.0, 6.6
#### browsers
Noticed via Chrome
#### OSes
Testing on Windows 10
### plugins
No plugins

**Comment by gkatsev** [comment]
I do this in the starter template test case which uses Video.js 5.x but cannot reproduce this in the 6.x player. Seems like in 6.x we are doing various null-checks to make sure we set the value properly and we set the initial value to be 0.
The 5.x player is mostly on maintenance right now so we're unlikely to get to this very soon. If you're interesting, would you be able to make a PR to add a null-check [here](https://github.com/videojs/video.js/blob/5.x/src/js/control-bar/progress-control/seek-bar.js#L99)?

**Comment by monicao** [comment]
Would this just be a matter of changing

```js
el.setAttribute('aria-valuenow', (this.getPercent() * 100).toFixed(2));
```

to 

```js
el.setAttribute('aria-valuenow', ((this.getPercent() || 0) * 100).toFixed(2));
```

?

**Comment by gkatsev** [comment]
That would probably work.

**Comment by gkatsev** [comment]
@monicao hey, were you still interested in making this change?

**Comment by monicao** [comment]
Yes, I'd love to, I'll submit a PR today!
